config/routes.rb

```diff
@@ -1,4 +1,35 @@
 Rails.application.routes.draw do
+
+  root "welcome#index"
+
+  get 'welcome/index'
+
+  resources :products, :only => [:index, :show] do
+    resources :order_items, :only => [:create, :update, :destroy]
+    resources :reviews
+    post 'add_to_cart', on: :member
+  end
+  get 'search' => 'products#search', as: 'search_product'
+
+  resources :users, :only => [:new, :create, :show, :edit, :update] do
+    resources :reviews
+    scope module: :users do
+      resources :products
+      resources :fulfillment, :only => [:index, :show]
+    end
+  end
+
+  resources :orders, :only => [:show, :edit, :update]
+
+  resources :order_items, :only => [:show]
+
+  resources :sessions, :only => [:create]
+  delete "/logout" => "sessions#destroy"
+  get    "/login"  => "sessions#new"
+
+  resources :categories, :only => [:new, :create]
+
+
```

Comment:
@roseaboveit , We had meant to do a routes audit, but just ran out of time! I've been trying to be better about this on recent projects.